### PR TITLE
fix up grip service so that it can be passed to sower

### PR DESCRIPTION
**Pull request by matthewpeterkort**
Abstracts grip service envar out of code base and into helm chart. Must add 
```
- name: GRIP_SERVICE_NAME
            valueFrom:
              configMapKeyRef:
                name: grip-values
                key: serviceName
```
to sower config env var section for it to work.

See updated PRS:
https://github.com/ACED-IDP/submission/pull/32
https://github.com/ACED-IDP/aced_etl_pod/pull/36

### CODE REVIEW

File: helm/grip/templates/deployment.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: local-grip
+  name: grip-deployment
```
Comment (quinnwai) [inline helm/grip/templates/deployment.yaml:4]: Does this need to be changed in the [README.md](https://github.com/ACED-IDP/gen3-helm/blob/671e040e5af0e86c2eb9c023b1025e6ec8acef99/helm/grip/README.md/#L3) as well as the [local-grip.conf](https://github.com/ACED-IDP/gen3-helm/blob/671e040e5af0e86c2eb9c023b1025e6ec8acef99/helm/revproxy/gen3.nginx.conf/local-grip.conf/#L8-L20)?

File: helm/mongodb/values.yaml [hunk 2]
```diff
@@ -30,7 +30,7 @@ global:
 nameOverride: ""
 ## @param fullnameOverride String to fully override mongodb.fullname template
 ##
-fullnameOverride: "mongodb-deployment"
+fullnameOverride: "local-mongodb"
```
Comment (quinnwai) [inline helm/mongodb/values.yaml]: why is this getting changed to local when local-grip is getting changed to grip-deployment?

**Review by quinnwai — COMMENTED** [review]
Added some comments! Also, what's the benefit of passing the grip service name through sower? Starting testing atm

**Comment by matthewpeterkort** [comment]
> Added some comments! Also, what's the benefit of passing the grip service name through sower? Starting testing atm

So that if helm changes, the code doesn't break

File: helm/grip/templates/deployment.yaml (hunk 1 above)
Comment (matthewpeterkort) [reply helm/grip/templates/deployment.yaml:4]: yes. Forgot to restart revproxy when testing. See https://github.com/ACED-IDP/gen3-helm/pull/67/commits/7a1e628d53b430bb0cbbb7b58f5d880f79a58f7f

File: helm/mongodb/values.yaml (hunk 2 above)
Comment (matthewpeterkort) [reply helm/mongodb/values.yaml]: It was originally called local-mongodb. Was an overzealous change in the last PR ended up causing errors when testing in this PR.

**Review by quinnwai — DISMISSED** [review]
cool vibes

**Review by quinnwai — APPROVED** [review]
hi hello

**Pull request merged**